docs/source/en/main_classes/tokenizer.md

```diff
@@ -77,6 +77,8 @@ loaded very simply into 🤗 transformers. Take a look at the [Using tokenizers
     - batch_decode
     - decode
     - encode
+    - add_bos_token
+    - add_eos_token
```

Comment:
@ArthurZucker is this the correct place for the docs or is there a different init doc location you are thinking of?